src/core/Form/Textarea/Textarea.tsx

```diff
@@ -22,15 +20,16 @@ const textareaClassNames = {
   fullWidth: `${baseClassName}--full-width`,
   textareaContainer: `${baseClassName}_textarea-element-container`,
   textarea: `${baseClassName}_textarea`,
+  resizeHorizontal: `${baseClassName}_textarea-resize--horizontal`,
+  resizeBoth: `${baseClassName}_textarea-resize--both`,
+  resizeNone: `${baseClassName}_textarea-resize--none`,
```

Comment:
This was good change (with all related changes) 👍

Reply:
Might be needed elsewhere as well, but gets complicated quickly when there are a lot of options.